src/poker/game/GamePlay.java

```diff
@@ -0,0 +1,545 @@
+        howMuchToCallBigBlind(player1, player2);
+        System.out.println("1 = call 2 = raise 3 = fold");
+        Scanner scanner = new Scanner(System.in);
+        int decision2 = scanner.nextInt();
+        if (decision2 == 1) {
+            upadePotStackAmountInHandBigBlindPlayer(amountToCallBigBlind(player1, player2), player1);
+            makeBoardDependsGamePhase();
+        } else if (decision2 == 2) {
+            makeRaiseBigBlind(player1, player2);
+            inPositionPlayerCantCheckPostFlop(player1, player2);
+        } else {
+            System.out.println(player1.getNamePlayer() + " folds");
+            System.out.println(player2.getNamePlayer() + " wins " + gameState.getPot() + "$");
+        }
+    }
+
+    public void inPositionPlayerCantCheckPostFlop(Player player1, Player player2) {
+        printPot();
+        howMuchToCallSmallBlind(player1, player2);
+        System.out.println(player2.getNamePlayer() + " decyduje");
+        System.out.println("1 = call 2 = raise 3 = fold");
+        Scanner scanner = new Scanner(System.in);
+        int decision2 = scanner.nextInt();
+        if (decision2 == 1) {
+            upadePotStackAmountInHandSmallBlindPlayer(amountToCallSmallBlind(player1, player2), player2);
+            makeBoardDependsGamePhase();
+        } else if (decision2 == 2) {
+            makeRaiseSmallBlind(player1, player2, amountToCallSmallBlind(player1, player2));
+            outPositionPlayerGotBetOrRaisePostFlop(player1, player2);
+        } else {
+            System.out.println(player2.getNamePlayer() + " folds");
+            System.out.println(player1.getNamePlayer() + " wins " + gameState.getPot() + "$");
+        }
+    }
+
+    //1111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111
+//1111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111
+//1111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111
+//1111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111
+//1111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111
+//1111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111
+//1111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111
+//1111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111
+//1111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111
+//1111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111
+//1111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111
+    public void preFlopSmallBlind(Player player1, Player player2) {
```

Comment:
niech sie to nazwy preflopAction

Reply:
zmieniles to na preflopActionSmall(...) raczej po bozemy powinna byc pelna nazwa preflopActionSmallBlind(...)